app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -340,9 +340,9 @@ class BookmarksTest {
         }
 
         multipleSelectionToolbar {
-        }.clickOpenNewTab {
+        }.clickOpenNewTab { }.openTabDrawer {
```

Comment:
Lets open an issue for ux flows we didn't account for and review with Nicole

UXFLOW